tests/diff/float_values_test.go

```diff
@@ -85,4 +85,50 @@ func TestFloatValues(t *testing.T) {
 	/*t.Run("FindAndModify", func(t *testing.T) {
 
 	})*/
+
+	t.Run("UpdateOne", func(t *testing.T) {
+		t.Parallel()
+
+		for name, tc := range map[string]struct {
+			filter   bson.D
+			insert   bson.D
+			update   bson.D
+			expected mongo.CommandError
+		}{
+			"MulMaxFloat64": {
+				filter: bson.D{{"_id", "number"}},
+				insert: bson.D{{"_id", "number"}, {"v", int32(42)}},
+				update: bson.D{{"$mul", bson.D{{"v", math.MaxFloat64}}}},
+				expected: mongo.CommandError{
+					Code:    2,
+					Name:    "BadValue",
+					Message: `invalid value: { "v": +Inf } (infinity values are not allowed)`,
```

Comment:
Ok, after looking at this case I'm not sure we should behave differently :)
"Infinity values are not allowed" is for the case when an input document has infinity in one of the fields. This case is about the result of the operation. 
So, we either should support it and return infinity (if it's possible in our code base), or it should be a different diff.md line for this documentation - https://docs.ferretdb.io/diff/. And I expect the error to be more clear that the result of the operation gives infinity.

Reply:
Example: we consider a filter with infinity acceptable, e.g. something like `{v: {$lt: +Inf}}` should be a supported input. So I wonder what should be our behaviour if Inifinity is in the output 🤔

Reply:
We probably don't want to store infinity values, only use them for comparison. So if we get an infinity value as the operation result we might want to return an error describing that behavior.